src/diffusers/utils/peft_utils.py

```diff
@@ -180,6 +180,14 @@ def set_adapter_layers(model, enabled=True):
                 module.disable_adapters = not enabled
 
 
+def delete_lora_layers(model, adapter_name):
```

Comment:
I think we should raise an error if PEFT version is not greater than 0.6.1

Reply:
Agree that a version check would be good to have. I would argue that all checks should be done in one place, i.e. either all inside of `delete_lora` or all here, but not part here part there.

Reply:
Yeah!

Reply:
Should this function also be renamed for consistency to `delete_adapter_layers`?

Reply:
Yes OK

Reply:
Done in [c54ec0c](https://github.com/huggingface/diffusers/pull/5738/commits/c54ec0cf30ba775cc7782af24a7793b418e9a0ee)